pytorch/training/docker/1.10/py3/Dockerfile.sagemaker.cpu

```diff
@@ -32,14 +32,21 @@ RUN conda install -y \
 
 RUN pip install --upgrade pip --trusted-host pypi.org --trusted-host files.pythonhosted.org \
  && pip install --no-cache-dir -U \
-    smdebug==${SMDEBUG_VERSION} \
+    # smdebug==${SMDEBUG_VERSION} \
     smclarify \
     "sagemaker>=2,<3" \
     sagemaker-experiments==0.* \
     "sagemaker-pytorch-training<3" \
  && pip uninstall -y torch \
  && pip install --no-cache-dir -U ${PT_TRAINING_URL}
 
+# Install smdebug from souce for test purpose
+RUN cd /tmp \
+  && git clone -b 1.0.13 https://github.com/awslabs/sagemaker-debugger \
```

Comment:
I am seeing that the SMDEBUG_VERSION used above is 1.0.12, however we are checking out to 1.0.13, is that on purpose?

Reply:
SMDEBUG team suggested to make such change. smdebug 1.0.12 does not support PT1.10.2, upgrading to 1.0.13 for the support

Reply:
I think it is a bit confusing to see `SMDEBUG_VERSION=1.0.12` and then cloning branch `1.0.13`. `SMDEBUG_VERSION` is not being used anywhere in the dockerfile. Why don't we change it to `1.0.13` and use it in the git clone command?

Reply:
I think smdebug team has not release 1.0.13 into PyPi. So the only way is to install from source code. This is also the way smdebug team suggested.

Reply:
Yes, I am agreeing to conduct the installation from the source code. I am just concerned that if we leave the `SMDEBUG_VERSION=1.0.12`, in the future, we might get confused by looking at this variable. So, we could make the following changes:
1. `SMDEBUG_VERSION=1.0.13`
2. git clone -b ${SMDEBUG_VERSION} https://github.com/awslabs/sagemaker-debugger

Reply:
Done